analyzer/runtime/incoming.go

```diff
@@ -523,21 +527,24 @@ func extractEvents(blockData *BlockData, relatedAccountAddresses map[apiTypes.Ad
 					}
 					amount := &big.Int{}
 					amount.SetBytes(amountU256)
+					ownerJSON, _ := json.Marshal(ethCommon.BytesToAddress(ownerEthAddr))
+					spenderJSON, _ := json.Marshal(ethCommon.BytesToAddress(spenderEthAddr))
+					amountJSON, _ := json.Marshal(amount)
```

Comment:
The PR description says the db stores the number correctly, so this is functionally a no-op? It's just because the `EvmEventParam` type changed, right?